config/config_test.go

```diff
@@ -305,7 +306,42 @@ func Test_Defaults(t *testing.T) {
 			path:          "Metrics.Enabled",
 			expectedValue: false,
 		},
-		// TODO(pg): add Aggregator section
+		{
+			path:          "Aggregator.Host",
+			expectedValue: "0.0.0.0",
+		},
+		{
+			path:          "Aggregator.Port",
+			expectedValue: 50081,
+		},
+		{
+			path:          "Aggregator.RetryTime",
+			expectedValue: types.NewDuration(5 * time.Second),
+		},
+		{
+			path:          "Aggregator.VerifyProofInterval",
+			expectedValue: types.NewDuration(90 * time.Second),
+		},
+		{
+			path:          "Aggregator.TxProfitabilityCheckerType",
+			expectedValue: aggregator.TxProfitabilityCheckerType(aggregator.ProfitabilityAcceptAll),
+		},
+		{
+			path:          "Aggregator.TxProfitabilityMinReward",
+			expectedValue: aggregator.TokenAmountWithDecimals{Int: big.NewInt(1100000000000000000)},
+		},
+		{
+			path:          "Aggregator.ProofStatePollingInterval",
+			expectedValue: types.NewDuration(5 * time.Second),
+		},
+		{
+			path:          "Aggregator.CleanupLockedProofsInterval",
+			expectedValue: types.NewDuration(10 * time.Minute),
```

Comment:
I think we can set CleanupLockedProofsInterval to 2 minutes and GeneratingProofCleanupThreshold to 10 minutes by default (for internal/public/main testnet). With these value in the worst scenario we will clean up a stucker proof in 12 minutes (from generating time). The provers used in production take less than 5-6 minutes to generate the batch proof (that is the most time consuming step).

On the other hand I see that we use "X * time.Second"  or "X * time.Minute to specify the interval value but in the case of the GeneratingProofCleanupThreshold we are using "30m", we do it by any reason? if not, to be consistent, I think we need to uses "30 * time.Minute". Anyway in my opinion is best to use the "30m" style for all the interval config values, but I think at least we need to specify the interval in the same way for all the config params.

If you agree with these values we need to change it also in the toml config file and in default.go file

Reply:
@agnusmor I modified the config parameters according to your proposed values: 642ad61555b779d4bc154e1102834fa71c7a9652

About the difference that you found in `config/config_test.go`, please note that this is a test case asserting that the parameters are correctly parsed by the Go package (we use `github.com/spf13/viper`). It does that by calling `config.Load`: https://github.com/0xPolygonHermez/zkevm-node/blob/642ad61555b779d4bc154e1102834fa71c7a9652/config/config_test.go#L356

Hence the difference on the Go side: `CleanupLockedProofsInterval` is type `types.Duration` whereas `GeneratingProofCleanupThreshold` is type `string`. This because `GeneratingProofCleanupThreshold` is not directly used in the Go code, instead it is further parsed into Postgres `interval` type: https://github.com/0xPolygonHermez/zkevm-node/blob/642ad61555b779d4bc154e1102834fa71c7a9652/state/pgstatestorage.go#L1985-L1989

But on the `.toml` files both params follows the same syntax as you can see: https://github.com/0xPolygonHermez/zkevm-node/blob/642ad61555b779d4bc154e1102834fa71c7a9652/test/config/test.node.config.toml#L92-L93